tests/agent_unittests/test_agent_protocol.py

```diff
@@ -170,7 +170,7 @@ def test_send(status_code):
     HttpClientRecorder.STATUS_CODE = status_code
     settings = finalize_application_settings(
         {
-            "request_headers_map": {"custom-header": u"value"},
+            "request_headers_map": {"custom-header": u"value"}, # pylint: disable=W1406
```

Comment:
Two spaces before inline comments:
```suggestion
            "request_headers_map": {"custom-header": u"value"},  # pylint: disable=W1406
```